municipal_finance/templates/table.html

```diff
@@ -23,7 +23,12 @@
     </div>
 
     <img src="{% static 'images/treasury-logo.png' %}" class="logo">
-    <h1><a href="/">Municipal Finance Data Tables</a></h1>
+    {% if site_notices %}
+      <h3><a href="/">Municipal Finance Data Tables</a></h3>
```

Comment:
This seems to be the top level heading on the page so it should use h1 not change to h3. Note CSS h3 styling above too.

Reply:
Changed the heading tag and corresponding CSS